cfme/tests/cli/test_appliance_console.py

```diff
@@ -729,36 +729,6 @@ def test_appliance_console_ha_setup_dc():
     pass
 
 
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_appliance_console_screen():
```

Comment:
@mnadeem92 the `screen` package is still present on 5.10.5.1 appliances, i.e. RHEL 7.6, shouldn't we still include this test case for 5.10?